frontend/src/screens/apps/NewApp.tsx

```diff
@@ -200,9 +200,13 @@ const NewAppInternal = ({ capabilities }: NewAppInternalProps) => {
         window.location.href = createAppResponse.returnTo;
         return;
       }
-      navigate(`/apps/created${app ? `?app=${app.id}` : ""}`, {
-        state: createAppResponse,
-      });
+      // Adding a unique id prevents https://github.com/getAlby/hub/issues/252
+      navigate(
+        `/apps/created?id=${createAppResponse.pairingPublicKey}${app ? `&app=${app.id}` : ""}`,
```

Comment:
It was already here, but why is there a conditional here? this seems wrong?

Also, the `pairingPublicKey` should be unique per app

Reply:
sorry, I completely misread this. I am wrong :cry:

Reply:
but it's weird, the navigation state should work. And it seems like it's not